sklearn/manifold/_spectral_embedding.py

```diff
@@ -330,7 +330,7 @@ def spectral_embedding(adjacency, *, n_components=8, eigen_solver=None,
             # lobpcg will fallback to eigh, so we short circuit it
             if sparse.isspmatrix(laplacian):
                 laplacian = laplacian.toarray()
-            _, diffusion_map = eigh(laplacian)
+            _, diffusion_map = eigh(laplacian, check_finite=False)
```

Comment:
this looks to be okay since it was directly checked by `check_array`.

Reply:
Thanks for the confirmation.